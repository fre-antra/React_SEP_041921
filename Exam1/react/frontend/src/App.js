import "./App.css";
import "./css/mainContent.css";
import "./css/navbar.css";
import MainContent from "./components/MainContent";
import NavBar from "./components/NavBar";
function App() {
    return (
        <div className="App">
            <NavBar />
            <MainContent />
        </div>
    );
}

export default App;
