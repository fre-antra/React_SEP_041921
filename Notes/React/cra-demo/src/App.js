import React  from 'react';

let data = {name: 'patrick'};

class App extends React.Component {
  state = {  data: {name: 'patrick'}}
  // constructor(props){
  //   super(props)
  //   this.state = {
  //     data:[{name: 'patrick'}]
  //   }
  // }

  render(){
    return <>
            <h1>Hello {this.state.data.name} {data.name}</h1>
            <button onClick ={() => {
              this.setState({data:{name:'Sam'}
              });
              }}>
                change click
            </button>
            <button 
              onClick ={ () => {
                data = {name:'Sam2'}
                console.log(data);
              }}
              >
                change click
            </button>
          
          </>
  }
}

export default App;
