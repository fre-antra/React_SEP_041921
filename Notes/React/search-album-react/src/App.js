import React, { useState } from "react";
import Header from "./components/Header";
import SearchGallery from "./components/SearchGallery";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import TopProfiles from "./components/TopProfiles";
//practice using contextAPI

export const MyContext = React.createContext();

function App() {
  const [searchInput, setSearchInput] = useState("");
  const handleSearchInput = input => {
    setSearchInput(input);
  };

  return (
    <Router>
      <div className="App">
        <MyContext.Provider value={{ searchInput, handleSearchInput }}>
          <Header />

          <Switch>
            <Route path="/" exact>
              <SearchGallery />
            </Route>
            <Route path="/top-profiles" exact>
              <TopProfiles />
            </Route>
          </Switch>
        </MyContext.Provider>
      </div>
    </Router>
  );
}
export default App;
