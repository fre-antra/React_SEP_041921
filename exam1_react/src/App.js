import logo from './logo.svg';
import NavBar from "./Components/NavBar";

import React, { Component } from 'react';
import SearchResult from "./Components/SearchResult";

export default class App extends Component {
  constructor(props) {
    super(props);
    this.state = {
      albumCount: 0,
      albumList: [],
      artistName: ""
    };
    this.searchArtist = this.searchArtist.bind(this);
  }

  searchArtist(artistName) {
    try {
      fetch(`https://itunes.apple.com/search?term=${artistName}&media=music&entity=album&attribute=artistTerm&limit=500`)
        .then(response => response.json())
        .then(result => {
          this.setState({
            albumCount: result.resultCount,
            albumList: result.results,
            artistName: artistName
          });
        });
    }
    catch (error) {
      console.log(error);
    }

  }
  render() {
    let { artistName, albumList, albumCount } = this.state;
    return (
      <div className="App">
        <NavBar searchArtist={this.searchArtist} />
        <SearchResult artistName={artistName} albumList={albumList} albumCount={albumCount} />
      </div>
    );
  }
}

