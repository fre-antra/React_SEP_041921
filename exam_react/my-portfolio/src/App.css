html {
    scroll-behavior: smooth;
  }
  
  /* Navigation */
  .navigation {
    position: absolute;
    width: 100%;
    padding: 10px 0px;
    background: linear-gradient(rgba(20, 20, 20, 0.1), rgba(20, 20, 20, 0.1));
  }
  
  .logo img {
    max-width: 90px;
  }
  .primary-nav {
    padding-right: 50px;
  }
  .primary-nav li {
    display: inline-block;
    margin: 0px 20px;
    color: white;
    font-size: 13pt;
  }
  
  .primary-nav a {
    color: white;
    padding-bottom: 5px;
    -webkit-transition: border 50ms linear;
    -moz-transition: border 50ms linear;
    -o-transition: border 50ms linear;
    -ms-transition: border 50ms linear;
    transition: border 50ms linear;
  }
  .primary-nav a:hover {
    text-decoration: none;
    color: white;
    border-bottom: 2px solid #f5f5f5;
  }
  
  /* Main banner */
  .main {
    background: linear-gradient(rgba(20, 20, 20, 0.7), rgba(20, 20, 20, 0.7)),
      url('./images/apple-1838564_1920-main.jpg'); /* Browser prefixes*/
    background-repeat: no-repeat;
    background-position: center center;
    background-attachment: fixed;
    padding: 14.7% 0; /* Center contents within the container, make background image cover*/
    color: white;
  }
  
  .entry-banner h1 {
    font-family: "Source Sans Pro", Arial, sans-serif;
  }
  #hello {
    font-size: 28pt;
  }
  #name {
    font-size: 100pt;
  }
  #person-description {
    font-size: 33pt;
  }
  .banner-buttons .btn-border,
  .submit-contact .btn-border,
  .projects .btn-border {
    background-color: transparent;
    width: 100%;
    margin-top: 40px;
    color: white;
    font-family: "Roboto", Helvetica, serif;
    font-size: 16pt;
    border-radius: 0;
    border: 2px solid white;
    -webkit-transition: all 200ms ease-out;
    -moz-transition: all 200ms ease-out;
    -o-transition: all 200ms ease-out;
    -ms-transition: all 200ms ease-out;
    transition: all 200ms ease-out;
  }
  
  .banner-buttons .btn-border:hover,
  .submit-contact .btn-border:hover,
  .projects .btn-border:hover {
    border: 2px solid white;
    background-color: white;
  }
  
  /* About Me */
  .about {
    padding: 3% 0;
    background: linear-gradient(
        rgba(225, 225, 225, 0.94),
        rgba(225, 225, 225, 0.94)
      ),
      url('./images/aboutbackground.jpg');
    background-repeat: no-repeat;
    background-attachment: fixed;
    background-position: center center;
  }
  
  .about-header h1 {
    font-family: "Source Sans Pro", Arial, sans-serif;
    font-size: 34pt;
  }
  
  .about-header-quote {
    font-family: "Slabo", Helvetica, serif;
    font-style: italic;
    margin-bottom: 50px;
  }
  
  .about-general-description,
  .about-statistic-description {
    display: flex;
    justify-content: center;
  }
  .about-general-description h2,
  .about-statistic-description h2 {
    font-family: "Source Sans Pro", Arial, sans-serif;
    font-size: 24pt;
    margin-bottom: 20px;
  }
  
  .about-general-description p,
  .about-statistic-description p {
    font-family: "Roboto", Helvetica, serif;
    font-size: 12pt;
    margin-bottom: 20px;
  }
  
  .about-statistic-description p {
    margin: 5px 0px;
  }
  .about-option-buttons {
    display: flex;
    justify-content: center;
  }
  .about-option-buttons .btn-black-border,
  .resume .btn-black-border {
    background-color: transparent;
    width: 100%;
    margin-top: 40px;
    color: black;
    font-family: "Roboto", Helvetica, serif;
    font-size: 16pt;
    border-radius: 0;
    border: 2px solid black;
    -webkit-transition: all 200ms ease-out;
    -moz-transition: all 200ms ease-out;
    -o-transition: all 200ms ease-out;
    -ms-transition: all 200ms ease-out;
    transition: all 200ms ease-out;
  }
  
  .about-option-buttons .btn-black-border:hover,
  .resume .btn-black-border:hover {
    border: 2px solid #464646;
    background-color: #464646;
    color: white;
  }
  
  .summary {
    background: linear-gradient(rgba(0, 0, 0, 0.9), rgba(0, 0, 0, 0.9)),
      url('./images/architecture-2178604_1920-main1.jpg');
    background-attachment: fixed;
    background-attachment: center center;
    color: white;
    padding: 60px 30px;
  }
  
  .summary .row {
    padding: 50px 6%;
    width: 100%;
  }
  
  .summary h3 {
    font-family: "Source Sans Pro", Arial, sans-serif;
  }
  .summary p {
    font-family: "Roboto", Helvetica, serif;
  }
  
  .projects {
    background-color: #59585d;
    padding: 50px;
    color: white;
  }
  
  .projects-header-quote {
    margin-bottom: 20px;
  }
  .projects h1,
  .projects h3 {
    font-family: "Source Sans Pro", Arial, sans-serif;
  }
  .projects p {
    font-family: "Roboto", Helvetica, serif;
    padding: 0 10px;
  }
  
  .projects-header-quote {
    font-style: italic;
  }
  /*.luvtalk-project, .personal-website-project, .strike-zone-project, .resume {border: 3px solid white;}*/
  .luvtalk-project {
    background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url('./images/luvtalklogo.png');
    background-position: center center;
  }
  
  .personal-website-project {
    background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url('./images/logo.svg');
    background-position: center center;
  }
  
  .strike-zone-project {
    background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url('./images/mlbproject.png');
    background-position: center center;
  }
  
  .resume {
    padding: 50px 0px;
    font-family: "Calisto MT", Helvetica, serif;
  }
  
  .resume-header {
    margin-bottom: 20px;
  }
  .objective h4,
  .education h4,
  .work-experience h4,
  .resume-projects h4,
  .activities h4,
  .skills h4,
  .endorsements h4 {
    border-bottom: 1px solid black;
  }
  .contact {
    background-color: #3a3a3a;
    padding: 50px 150px;
    color: white;
  }
  .contact input,
  textarea {
    color: black;
  }
  
  .contact h1,
  .contact h2,
  .contact h4 {
    font-family: "Source Sans Pro", Arial, sans-serif;
  }
  .contact p {
    font-family: "Roboto", Helvetica, serif;
  }
  .contact-header-quote {
    font-style: italic;
  }
  .contact-header-quote h4 {
    margin: 5px 0;
  }
  .contact-instructions,
  .contact-form-body {
    margin-top: 50px;
    padding: 0px 6%;
  }
  
  .email,
  .message {
    margin: 15px 0px;
  }
  .contact input,
  .contact textarea {
    width: 100%;
    padding: 10px;
    margin: 3px 0px;
  }
  
  .submit-contact .btn-border {
    margin-top: 10px;
  }
  .footer {
    background-color: black;
    color: white;
    padding: 50px 0;
    font-family: "Roboto", Helvetica, serif;
  }
  
  .footer h3,
  .footer h4 {
    font-family: "Source Sans Pro", Arial, sans-serif;
  }
  .facebook,
  .linkedin,
  .pinterest,
  .email-icon {
    padding: 31%;
  }
  .footer img {
    background-color: #8a8a8a;
    -webkit-transition: all 200ms ease-out;
    -moz-transition: all 200ms ease-out;
    -o-transition: all 200ms ease-out;
    -ms-transition: all 200ms ease-out;
    transition: all 200ms ease-out;
  }
  
  .footer img:hover {
    background-color: white;
  }
  .company-information,
  .home-footer,
  .portfolio-footer,
  .resume-footer,
  .contact-footer {
    margin-top: 50px;
  }
  .footer ul {
    list-style: none;
    padding-left: 0;
  }
  
  .home-footer a,
  .portfolio-footer a,
  .resume-footer a,
  .contact-footer a {
    color: white;
  }
  .home-footer a:hover,
  .portfolio-footer a:hover,
  .resume-footer a:hover,
  .contact-footer a:hover {
  }
  