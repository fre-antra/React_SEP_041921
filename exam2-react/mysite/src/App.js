import "./App.css";
import "./bootstrap/css/bootstrap.min.css";
import "./style.css";
import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
import { useEffect, useRef, useState } from "react";

import Footer from "./components/Footer";
import Contact from "./components/Contact";
import Portfolio from "./components/Portfolio";
import Navbar from "./components/Navbar";
import Main from "./components/Main";

function App() {
  const ref = useRef();

  const [domState, setDOMState] = useState(); //state hold ref to dom element to be passed as props
  const setTargetRef = reference => {
    setDOMState(reference);
  };

  return (
    <Router>
      <div className="App">
        <Navbar />
        <Switch>
          <Route path="/" exact>
            <Main targetRef={domState} />
            <Portfolio refProp={ref} setTargetRef={setTargetRef} />
            <Contact />
          </Route>
          <Route path="/portfolio" exact>
            <Portfolio />
          </Route>
          <Route path="/contact" exact>
            <Contact />
          </Route>
        </Switch>
        <Footer />
      </div>
    </Router>
  );
}

export default App;
