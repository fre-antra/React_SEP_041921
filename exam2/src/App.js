import logo from './logo.svg';
import './App.css';
import React from 'react';
import Layout from './components/Layout/Layout';
import { BrowserRouter, Route, Link } from 'react-router-dom';
import Contact from './components/Contact/Contact'
import Hometop from './components/Hometop/Hometop'
import Portfolio from './components/Portfolio/Portfolio'

class App extends React.Component {

  render() {
    return (
      <BrowserRouter>
        <Layout>
          <Route exact path="/home" render={() =>
            <>
              <Hometop />
              <Portfolio />
              <Contact />
            </>
          } />

          <Route exact path="/portfolio" component={Portfolio}></Route>
          <Route exact path="/contact" component={Contact}></Route>
        </Layout>
      </BrowserRouter>
    );
  }
}

export default App;
