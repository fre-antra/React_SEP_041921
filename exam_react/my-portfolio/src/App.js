import React, { Component, Fragment } from 'react'
import { BrowserRouter as Router, Route } from "react-router-dom";
import Layout from './components/Layout/index';
// import Navbar from './components/Navbar';
import Home from './components/Home/index';
import Portfolio from './components/Portfolio/index';
import Contact from './components/Contact/index';
import './App.css'

class App extends Component {
  render() {
    return (
      <Fragment>
        <Layout>
          <Router>
            <Route path="/portfolio" component={Portfolio} />
            <Route path="/contact" component={Contact} />
            <Route exact path="/" component={Home} />
          </Router>
        </Layout>
      </Fragment>
    )
  }
}

export default App;
