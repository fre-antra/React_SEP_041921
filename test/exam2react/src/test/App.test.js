import { render, fireEvent, waitFor, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import App from "../App";

describe("App component", () => {
  it("render without error", () => {
    const { container } = render(<App></App>);
    expect(container).toMatchSnapshot();
  });

  it('has Nav component', () => {
    render(<App></App>);
    const navbar = screen.getByTestId('navbar');
    expect(navbar).toBeInTheDocument()
  })

  it('has Footer component', () => {
    render(<App></App>);
    const footer = screen.getByTestId('footer');
    expect(footer).toBeInTheDocument()
  })
});
