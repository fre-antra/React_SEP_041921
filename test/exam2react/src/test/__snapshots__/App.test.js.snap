// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component render without error 1`] = `
<div>
  <div
    class="App"
  >
    <div
      data-testid="navbar"
    >
      <div>
        <nav
          class="navbar"
        >
          <img
            alt="logo"
            class="navbar__logo"
            src="logo.svg"
          />
          <ul
            class="navbar__list"
          >
            <li>
              <a
                class="navbar__list-item"
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                class="navbar__list-item"
                href="/portfolio"
              >
                Portfolio
              </a>
            </li>
            <li>
              <a
                class="navbar__list-item"
                href="/contact"
              >
                Contact
              </a>
            </li>
          </ul>
        </nav>
        <div
          data-testid="home-section"
          id="home"
        >
          <section
            class="hero-section"
          >
            <div
              class="hero-section__background"
            >
              <div
                class="react-parallax "
                style="position: relative; overflow: hidden;"
              >
                <img
                  alt=""
                  class="react-parallax-bgimage"
                  src="heroimage.jpg"
                  style="position: absolute; left: 50%; transform: translate3d(-50%, 0, 0); opacity: 0.9;"
                />
                <div
                  class="react-parallax-content"
                  style="position: relative;"
                >
                  <div
                    class="dark-layer text-center"
                  >
                    <div
                      class="center-div undefined"
                      style="height: 100vh; width: 100%;"
                    >
                      <p
                        class="hero-section__intro"
                      >
                        Hello I'm
                      </p>
                      <p
                        class="hero-section__name"
                      >
                        XXXX
                      </p>
                      <p
                        class="hero-section__jobtitle"
                      >
                        UI Developer. UX Desinger. Problem Solver.
                      </p>
                      <div>
                        <button
                          class="btn__ghost-btn"
                        >
                          Info
                        </button>
                        <button
                          class="btn__ghost-btn"
                        >
                          Portfolio
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </section>
          <section
            class="aboutme-section"
          >
            <div
              class="react-parallax "
              style="position: relative; overflow: hidden;"
            >
              <img
                alt="aboutMe"
                class="react-parallax-bgimage"
                src="aboutme.jpg"
                style="position: absolute; left: 50%; transform: translate3d(-50%, 0, 0); opacity: 0.9;"
              />
              <div
                class="react-parallax-content"
                style="position: relative;"
              >
                <div
                  class="white-layer"
                >
                  <div
                    class="center-div undefined"
                  >
                    <p
                      class="aboutme-section__intro"
                    >
                      About me
                    </p>
                    <i
                      class="aboutme-section__subintro"
                    >
                      Sharing a little bit of my story...
                    </i>
                  </div>
                  <div
                    class="ui container pl-60 pr-60"
                  >
                    <p
                      class="aboutme-section__header"
                    >
                      General Overview
                    </p>
                    <p
                      class="aboutme-section__text"
                    >
                      Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum
                    </p>
                    <p
                      class="aboutme-section__header"
                    >
                      Statistics
                    </p>
                    <i
                      class="aboutme-section__text"
                    >
                      And for those that don't fancy reading:
                    </i>
                    <p
                      class="aboutme-section__text"
                    >
                      Male, 200 years old. 
                      <br />
                      Five feet 110 inches 
                      <br />
                      Favorite fonts:Roboto 
                      <br />
                      Loves to learn new things
                    </p>
                    <div
                      class="center-div undefined"
                    >
                      <button
                        class="btn__ghost-btn--black mb-50"
                      >
                        <a
                          href="#contact"
                        >
                          Contact
                        </a>
                      </button>
                      <div
                        class="mb-50"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </section>
          <section
            alt="responsive"
            class="responsive-section"
          >
            <div
              class="react-parallax "
              style="position: relative; overflow: hidden;"
            >
              <img
                alt=""
                class="react-parallax-bgimage"
                src="architecture-2178604_1920-main1.jpg"
                style="position: absolute; left: 50%; transform: translate3d(-50%, 0, 0); opacity: 0.9;"
              />
              <div
                class="react-parallax-content"
                style="position: relative;"
              >
                <div
                  class="center-div__column-row  pl-50 pr-50 pt-60 pb-60 "
                  style="background-color: black; opacity: 0.8; min-height: 600px;"
                >
                  <div
                    class="ui centered grid"
                  >
                    <div
                      class="row"
                    >
                      <div
                        class="seven wide computer sixteen wide mobile eight wide tablet column"
                      >
                        <div
                          class="center-div__column-row undefined"
                        >
                          <div
                            class="ui grid"
                          >
                            <div
                              class="three wide computer sixteen wide mobile six wide tablet column"
                            >
                              <img
                                class="responsive-section__image"
                                src="developerdesign.png"
                              />
                            </div>
                            <div
                              class="ten wide computer sixteen wide mobile ten wide tablet column"
                            >
                              <div>
                                <p
                                  class="responsive-section__header-text"
                                >
                                  Development and Design
                                </p>
                                <p
                                  class="responsive-section__body-text"
                                >
                                  I aim to put my creativity to the test, designing and building unique, meaningful products for clients or merely for my own interests.
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        class="seven wide computer sixteen wide mobile eight wide tablet column"
                      >
                        <div
                          class="center-div__column-row undefined"
                        >
                          <div
                            class="ui grid"
                          >
                            <div
                              class="three wide computer sixteen wide mobile six wide tablet column"
                            >
                              <img
                                class="responsive-section__image"
                                src="responsivedesign.png"
                              />
                            </div>
                            <div
                              class="ten wide computer sixteen wide mobile ten wide tablet column"
                            >
                              <div>
                                <p
                                  class="responsive-section__header-text"
                                >
                                  Responsive Layouts
                                </p>
                                <p
                                  class="responsive-section__body-text"
                                >
                                  Development and design isn't merely putting information on the site or preferred media outlet. I organize content and present information in an engaging fashion, driving new and unique layouts in tandem with novel solutions and cool animations.
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      class="mb-50"
                    />
                    <div
                      class="row"
                    >
                      <div
                        class="seven wide computer sixteen wide mobile eight wide tablet column"
                      >
                        <div
                          class="center-div__column-row undefined"
                        >
                          <div
                            class="ui grid"
                          >
                            <div
                              class="three wide computer sixteen wide mobile six wide tablet column"
                            >
                              <img
                                class="responsive-section__image"
                                src="innovativesolutions.png"
                              />
                            </div>
                            <div
                              class="ten wide computer sixteen wide mobile ten wide tablet column"
                            >
                              <div>
                                <p
                                  class="responsive-section__header-text"
                                >
                                  Ideas and Solutions
                                </p>
                                <p
                                  class="responsive-section__body-text"
                                >
                                  There are still many problems that exist in today's society, including laziness. Luckily, I hope to combat these issues by innovating, developing easy-to-use programs, solutions, or products.
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        class="seven wide computer sixteen wide mobile eight wide tablet column"
                      >
                        <div
                          class="center-div__column-row undefined"
                        >
                          <div
                            class="ui grid"
                          >
                            <div
                              class="three wide computer sixteen wide mobile six wide tablet column"
                            >
                              <img
                                class="responsive-section__image"
                                src="passion.png"
                              />
                            </div>
                            <div
                              class="ten wide computer sixteen wide mobile ten wide tablet column"
                            >
                              <div>
                                <p
                                  class="responsive-section__header-text"
                                >
                                  Passion and Dedication
                                </p>
                                <p
                                  class="responsive-section__body-text"
                                >
                                  With my profound interest and commitment to my field of study, my projects rarely go unfinished and my problems are never left unresolved.
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </section>
          <section
            id="portfolio"
          >
            <div
              class="portfolio mb-60"
            >
              <div
                class="pt-60"
              />
              <div
                class="center-div undefined"
              >
                <p
                  class="portfolio-section__intro"
                  data-testid="portfolio-header"
                >
                  Projects and Portfolio
                </p>
                <i
                  class="portfolio-section__subintro mb-50"
                >
                  Sharing my endeavors and passions...
                </i>
                <div
                  class="mb-50"
                />
                <div
                  class="ui container three column grid"
                >
                  <div
                    class="five wide computer sixteen wide mobile eight wide tablet column"
                  >
                    <div>
                      <div
                        class="item"
                        style="background-image: url(luvtalklogo.png); width: 100%; background-position: center; background-size: cover;"
                      >
                        <div
                          class="dark-layer item-layer"
                          style="width: 100%;"
                        >
                          <div
                            class="center-div undefined"
                          >
                            <p
                              class="portfolio-section__header"
                            >
                              LUV TALK Website
                            </p>
                            <div
                              class="mb-50"
                              style="width: 90%; text-align: center;"
                            >
                              <p>
                                Website, created during my first internship, which utilized HTML, CSS, PHP, Ajax, Javascript/jQuery, and Wordpress.
                              </p>
                            </div>
                            <button
                              class="btn__ghost-btn mb-50"
                            >
                              <a
                                href="#"
                              >
                                More
                              </a>
                            </button>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="five wide computer sixteen wide mobile eight wide tablet column"
                  >
                    <div>
                      <div
                        class="item"
                        style="background-image: url(logo.svg); width: 100%; background-position: center; background-size: cover;"
                      >
                        <div
                          class="dark-layer item-layer"
                          style="width: 100%;"
                        >
                          <div
                            class="center-div undefined"
                          >
                            <p
                              class="portfolio-section__header"
                            >
                              Personal Website
                            </p>
                            <div
                              class="mb-50"
                              style="width: 90%; text-align: center;"
                            >
                              <p>
                                Enjoyable side project that was created to experiment with more HTML and CSS, but also provided an outlet to showcase my abilities and interests.
                              </p>
                            </div>
                            <button
                              class="btn__ghost-btn mb-50"
                            >
                              <a
                                href="#"
                              >
                                More
                              </a>
                            </button>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="five wide computer sixteen wide mobile eight wide tablet column"
                  >
                    <div>
                      <div
                        class="item"
                        style="background-image: url(mlbproject.png); width: 100%; background-position: center; background-size: cover;"
                      >
                        <div
                          class="dark-layer item-layer"
                          style="width: 100%;"
                        >
                          <div
                            class="center-div undefined"
                          >
                            <p
                              class="portfolio-section__header"
                            >
                              Strike Zone Analysis
                            </p>
                            <div
                              class="mb-50"
                              style="width: 90%; text-align: center;"
                            >
                              <p>
                                Data analytics project completed during my time at the Illinois Math and Science Academy which studied the baseball strike zone based on the state of the game.
                              </p>
                            </div>
                            <button
                              class="btn__ghost-btn mb-50"
                            >
                              <a
                                href="#"
                              >
                                More
                              </a>
                            </button>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="mt-60"
                />
                <button
                  class="btn__ghost-btn "
                >
                  <a
                    href="#"
                  >
                    More Projects
                  </a>
                </button>
                <div
                  class="mb-50"
                />
              </div>
            </div>
          </section>
          <section
            id="contact"
          >
            <div
              class="contact pl-60 pr-60"
            >
              <div
                class="center-div undefined"
              >
                <p
                  class="contact-section__intro text-center"
                  data-testid="contact-header"
                >
                  Contact
                </p>
                <i
                  class="contact-section__subintro text-center"
                >
                  "XXXXXXXXXXX"
                  <br />
                  -XXXXXXXXXXXX
                </i>
                <div
                  class="ui container pr-60 pl-60"
                  style="width: 80%;"
                >
                  <div
                    class="ui centered middle aligned two column grid"
                  >
                    <div
                      class="row"
                    >
                      <div
                        class="eight wide computer sixteen wide mobile eight wide tablet column"
                      >
                        <p
                          class="contact-section__header"
                        >
                          Reaching out to me
                        </p>
                        <p>
                          Don't be afraid to contact me! I'm happy to answer any questions, provide more information, or just have a nice conversation! Fill out the form below to being reaching out to me. If you prefer another email client other than your default, which will appear after clicking the button and have all of the information you just entered, you can email me at XXXXX.
                        </p>
                        <p
                          class="contact-section__header"
                        >
                          Contact Information
                        </p>
                        <p>
                          XXXX 
                          <br />
                           XXXX 
                          <br />
                           (XXXX)-445-7747 
                          <br />
                           
                          XXXXXX@gmail.com
                        </p>
                      </div>
                      <div
                        class="eight wide computer sixteen wide mobile eight wide tablet column"
                      >
                        <div
                          class="mt-50"
                        />
                        <form>
                          <div
                            class="ui grid"
                          >
                            <div
                              class="eight wide computer sixteen wide mobile eight wide tablet column"
                            >
                              <p
                                class="contact-section__form-title"
                              >
                                First Name *
                              </p>
                              <input
                                class="contact-section__form-input"
                              />
                            </div>
                            <div
                              class="eight wide computer sixteen wide mobile eight wide tablet column"
                            >
                              <p
                                class="contact-section__form-title"
                              >
                                Last Name *
                              </p>
                              <input
                                class="contact-section__form-input"
                              />
                            </div>
                            <div
                              class="sixteen wide computer sixteen wide mobile sixteen wide tablet column"
                            >
                              <p
                                class="contact-section__form-title"
                              >
                                Email *
                              </p>
                              <input
                                class="contact-section__form-input"
                              />
                            </div>
                            <div
                              class="sixteen wide computer sixteen wide mobile sixteen wide tablet column"
                            >
                              <p
                                class="contact-section__form-title"
                              >
                                Message *
                              </p>
                              <input
                                class="contact-section__form-input"
                              />
                            </div>
                          </div>
                        </form>
                        <div
                          class="center-div undefined"
                        >
                          <div
                            class="mt-60"
                          />
                          <button
                            class="btn__ghost-btn mb-50 "
                          >
                            <a
                              href="#"
                            >
                              Submit
                            </a>
                          </button>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </section>
        </div>
      </div>
    </div>
    <div
      class="footer-section"
      data-testid="footer"
    >
      <div
        class="center-div undefined"
      >
        <div
          class="mt-50"
        />
        <div
          class="ui centered grid"
        >
          <div
            class="center-div undefined"
          >
            <div
              class="ui grid"
            >
              <div
                class="row"
              >
                <div
                  class="four wide computer sixteen wide mobile column"
                >
                  <img
                    alt="logo"
                    class="icon"
                    data-testid="1"
                    src="facebook.svg"
                  />
                </div>
                <div
                  class="four wide computer sixteen wide mobile column"
                >
                  <img
                    alt="logo"
                    class="icon"
                    data-testid="2"
                    src="linkedin.svg"
                  />
                </div>
                <div
                  class="four wide computer sixteen wide mobile column"
                >
                  <img
                    alt="logo"
                    class="icon"
                    data-testid="3"
                    src="pinterest.svg"
                  />
                </div>
                <div
                  class="four wide computer sixteen wide mobile column"
                >
                  <img
                    alt="logo"
                    class="icon"
                    data-testid="4"
                    src="email.svg"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="ui centered grid"
            >
              <div
                class="eight wide computer eight wide mobile eight wide tablet column"
              >
                <p>
                  Phone: XXXXXX
                </p>
              </div>
              <div
                class="eight wide computer eight wide mobile eight wide tablet column"
              >
                <p>
                  Email: XXXXX@gmail.com
                </p>
              </div>
            </div>
          </div>
          <div
            class="centered row"
          >
            <div
              class="center-div undefined"
            >
              <div
                class="ui grid"
                data-testid="grid"
              >
                <div
                  class="three wide computer sixteen wide mobile column"
                >
                  <div
                    class="footer-section__list"
                  >
                    <a
                      class="footer-section__list-header"
                    >
                      XXXXXX
                    </a>
                    <a
                      href="#"
                    >
                      Copyright © 2021
                    </a>
                    <a
                      href="#"
                    >
                      NY
                    </a>
                  </div>
                </div>
                <div
                  class="three wide computer sixteen wide mobile column"
                >
                  <div
                    class="footer-section__list"
                  >
                    <a
                      class="footer-section__list-header"
                      href="#home"
                    >
                      Home
                    </a>
                  </div>
                </div>
                <div
                  class="three wide computer sixteen wide mobile column"
                >
                  <div
                    class="footer-section__list"
                  >
                    <a
                      class="footer-section__list-header"
                      href="#portfolio"
                    >
                      Portfolio
                    </a>
                    <a
                      href="#"
                    >
                      Project 1
                    </a>
                    <a
                      href="#"
                    >
                      Project 2
                    </a>
                    <a
                      href="#"
                    >
                      Project 3
                    </a>
                    <a
                      href="#"
                    >
                      See All
                    </a>
                  </div>
                </div>
                <div
                  class="three wide computer sixteen wide mobile column"
                >
                  <div
                    class="footer-section__list"
                  >
                    <a
                      class="footer-section__list-header"
                      href="#portfolio"
                    >
                      Resume
                    </a>
                    <a
                      href="#home"
                    >
                      Download
                    </a>
                  </div>
                </div>
                <div
                  class="three wide computer sixteen wide mobile column"
                >
                  <div
                    class="footer-section__list"
                  >
                    <a
                      class="footer-section__list-header"
                      href="#contact"
                    >
                      Contact
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
